src/app/navigation/example/application-launcher-example.component.html

```diff
@@ -0,0 +1,52 @@
+<div class="padding-15">
+  <div class="row">
+    <div class="col-sm-12">
+      <h4>Application Launcher</h4>
+      <hr/>
+    </div>
+
+    <div class="col-sm-8">
+      <nav class="navbar navbar-default navbar-pf" role="navigation">
+        <div class="navbar-header">
+          <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse-1">
+            <span class="sr-only">Toggle navigation</span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+          </button>
+          <a class="navbar-brand active" id="logo" routerlink="/" routerlinkactive="active" title="" ng-reflect-router-link="/" ng-reflect-router-link-active="active" href="#/">PatternFly NG</a>
+        </div>
+        <nav class="collapse navbar-collapse">
+          <ul class="nav navbar-nav navbar-right navbar-iconic navbar-utility">
+            <li class="dropdown">
+              <pfng-application-launcher [label]="'Application Launcher'" [isDisabled]="false" [isList]="false" [items]="navigationItems"
+                [hiddenIcons]="false">
+              </pfng-application-launcher>
+            </li>
+          </ul>
+        </nav>
+      </nav>
+    </div>
+  </div>
```

Comment:
The app launcher doesn't appear vertically centered like the Angular Patternfly example. Not sure if it's the height of the example (may be taller than angular-patternfly) or the component's CSS isn't vertically centered?

patternfly-ng:
![screen shot 2018-01-29 at 10 38 36 am](https://user-images.githubusercontent.com/17481322/35518934-e6bf823a-04e0-11e8-8d0f-459920a686c9.png)

angular-patternfly:
![screen shot 2018-01-29 at 10 38 46 am](https://user-images.githubusercontent.com/17481322/35518954-f2681ef8-04e0-11e8-9e93-cc4588883909.png)